generators/client-2/templates/src/main/webapp/app/blocks/interceptor/_http.interceptor.ts

```diff
@@ -31,7 +31,7 @@ export class HttpInterceptor extends Http {
     }
 
     request(url: string | Request, options?: RequestOptionsArgs): Observable<Response> {
-        return this.intercept(super.request(url, this.getRequestOptionArgs(options)));
+        return super.request(url, this.getRequestOptionArgs(options));
```

Comment:
Reasoning behind : Everytime we call http.get or smthing, this method (request) was called so we intercepted the same request twice. It doesn't make any error appear but I thought that was kinda useless to do the same thing twice ?